.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Magento Cloud Tools code
+
+Use the GitHub fork & pull model contribution model to submit your code contributions to the Magento Cloud Tools codebase.
+In this contribution model, you maintain your own [fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks) of the Magento Cloud Tools repository, and create a [pull request](https://help.github.com/articles/about-pull-requests/) to submit your proposed changes to the base repository. For details on the fork & pull contribution model, see the [Beginners guide](https://github.com/magento/magento2/wiki/Getting-Started).
```

Comment:
"Magento Cloud Tools"  ---  Magento Cloud ECE-Tools repository -- suggesting that to avoid confusion by adding another new name.